operations/onboarding.md

```diff
@@ -0,0 +1,111 @@
+
+### Due Diligence (Incubating and Graduating)
+
+The primary work a TOC member engages is in conducting the due diligence of cloud native projects as they advance in their maturity, this process is commonly referred to the moving levels or maturation process. After a TOC member steps forward to sponsor a project, it is anticipated that a kick-off meeting is held to set expectations for what the process will be like, provide rough timelines, collect additional information and allow the project to ask any questions.  This kick-off is commonly conducted as a virtual meeting, not lasting more than an hour. Additionally, TOC members should anticipate a few hours of concentrated review, analysing the project, its repository(ies), and clarifying the content of the due diligence. Additional time is expected to be spent conducting and summarizing Adopter interviews to ascertain the current usage of the project within the industry.
+
+### Health and alignment
+
+The TOC may choose to review a project as a result of a health indicator, project event, or other occassion. These reviews are typically specific to the occassion that warranted them but usually involves, researching the problem or indicator, understanding if the metric or occassion is accurate in its depiction and warrants attention, action, or other next step.
+
+When a health or alignment concern is elevated to the TOC, TOC members should adhere to the [project health review guide](project-health-review.md).
+
+## Voting
+
+The TOC primarily votes in one of two mechanisms, gitvote or on the mailing list. It is our preference that voting occur with [gitvote](https://github.com/cncf/gitvote) in the repository as this provides better transparency and is easier to verify and track vote execution. Voting conducted over the mailing list is performed by responding with a +1 Binding (+1B), or -1 Binding (-1). TOC members may abstain if they have a conflict of interest. Indication of abstention from voting for any other reason is discouraged and should not be confused with not yet voting as indication of abstention is active, not passive, and results in no vote being rendered by that individual. In the absence of information, we cannot make an informed decision, so it is imperative that we ask questions openly and provide answers with clarity for our peer TOC members, particularly on topics that may not be their strength. A vote passes with two-thirds vote of votes cast based on the charter (see [6(c)(viii)](https://www.cncf.io/about/charter)). In the rare event that an issue requiring a vote is senstive, the TOC will leverage the private mailing list to conduct the vote with the resulting decision made public.
```

Comment:
Any reason why we need 2 mechanisms? I agree gitvote is easier and I should have send Nic Jackson's tag-network co-chair as gitvote if I knew this earlier.

Reply:
it developed that way. Prior, we used the mailing list to for conducting voting. A lot of people are on the mailing list and provides an opportunity for subscribers to support those changes and nominations.  Gitvote however makes it much cleaner for the TOC to make decisions on specific items asynchronously, there is less manually counting involved and votes are not lost as we can see whose vote is still outstanding. Its more transparent.

We should always have a redundant system incase the primary doesnt work. Albeit we should consider what types of decisions warrant a mailing list vote versus notification and how we can redirect traffic to the TOC repo where those decisions are reflected.

Reply:
I think we should make sure that we are flexible with voting as long as it's transparent. I want to avoid projects voting on private mailing lists which I see sometimes and we should lead with example.

I do think GitVote makes things super easy and we should use it for more TOC activities